features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Resources/Resources.jsx

```diff
@@ -297,6 +297,12 @@ export default function Resources(props) {
             setOperationRateLimits(response.body.list);
         });
         // TODO: need to handle the error cases through catch ~tmkb
+        // if (api.isSOAPToREST()) {
+        //     api.getResourcePolicies().then((res) => {
+        //         const { a } = res;
+        //         console.log(a);
+        //     });
+        // }
```

Comment:
commented?

Reply:
Removed in https://github.com/wso2/carbon-apimgt/pull/7740/commits/cd50ac4ef19324d9f4ee4f3b6dc01cb590aeb3be